How to set correctly ssl certificate oauth2

Hi folks,

I'm trying to use oauth2 following this guide : https://quarkus.io/guides/security-oauth2 and also https://quarkus.io/guides/native-and-ssl

In application.properties, i use for the `quarkus.oauth2.introspection-url` an **https** adress ([auth0](https://auth0.com/)).

When i execute these commands separatly : 

```
./mvnw compile quarkus:dev
./mvnw clean install -Pnative 
```

I getting this error : 
> Caused by: java.lang.IllegalArgumentException: Parameter 'sslContext' may not be null

The complete stacktrace : [error-log.txt](https://github.com/quarkusio/quarkus/files/3884061/error-log.txt)

I use graalVM, the `GRAALVM_HOME` is set and the certificate of auth0 is added to the truststore of graalVM, follwing these commands :

```
echo | openssl s_client -servername NAME -connect HOST:PORT |\
  sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' > certificate.crt

/home/xerxes/graalvm-ce-19.2.1/bin/keytool -import -alias auth0 -storepass 'changeit' -keystore /home/xerxes/graalvm-ce-19.2.1/jre/lib/security/cacerts -file certificate.crt
```
I check with this command `keytool -list -v -keystore /home/xerxes/graalvm-ce-19.2.1/jre/lib/security/cacerts`, i have my certificate in there.

Could you explain me what i missed in my steps ?

I saw this option `quarkus.oauth2.ca-cert-file` but i don't know very well how to use it.

Thank you so much guys for your help !

--- a/extensions/elytron-security-oauth2/runtime/src/main/java/io/quarkus/elytron/security/oauth2/runtime/OAuth2Recorder.java
+++ b/extensions/elytron-security-oauth2/runtime/src/main/java/io/quarkus/elytron/security/oauth2/runtime/OAuth2Recorder.java
@@ -39,6 +39,8 @@ public class OAuth2Recorder {
 
         if (config.caCertFile.isPresent()) {
             validatorBuilder.useSslContext(createSSLContext(config));
+        } else {
+            validatorBuilder.useSslContext(SSLContext.getDefault());
         }
 
         OAuth2IntrospectValidator validator = validatorBuilder.build();

